pkg/operator/config.go

```diff
@@ -46,7 +46,7 @@ type Config struct {
 	SecretListWatchSelector      string
 }
 
-type ReloaderConfig struct {
+type ReloaderOperatorConfig struct {
```

Comment:
this struct is about configuration options for the container.
```suggestion
type ContainerConfig struct {
```